CoreDataAccessView/test/unit/src/au/gov/asd/tac/constellation/views/dataaccess/templates/FilterPluginNGTest.java

```diff
@@ -32,6 +39,24 @@
  * @author arcturus
  */
 public class FilterPluginNGTest {
+    
+    private static final Logger LOGGER = Logger.getLogger(FilterPluginNGTest.class.getName());
+    
+    @BeforeClass
+    public void setUpClass() throws Exception {
+        if (!FxToolkit.isFXApplicationThreadRunning()) {
+            FxToolkit.registerPrimaryStage();
```

Comment:
Curious as to the reason for registering the fx toolkit here (and in a few other unit test classes)? You normally only need to do that when dealing with stuff on the fx thread which is only needed when dealing with fx objects. Given there aren't any fx objects being used, it seems unnecessary (the tests certainly didn't need them previously)

Reply:
I was under the impression that this was a magical bit of code that made tests miraculously pass online and needet to be included in every test. I have since been educated that it is only relevant for tests involving JavaFX components so have removed it from these non-applicable tests :)